src/pages/api/timeRestrictions.ts

```diff
@@ -25,16 +29,23 @@ export default (req: NextApiRequestWithSession, res: NextApiResponse): void => {
         const { timeRestrictions } = req.body;
 
         if (timeRestrictions === 'no') {
+            updateSessionAttribute(req, TIME_RESTRICTIONS_ATTRIBUTE, {
+                timeRestrictions: false,
+            });
```

Comment:
Whats the reason for including this?

Reply:
I've added this in to allow us to render the breadcrumbs correctly. Could've implemented this so that the attribute is only set when `timeRestrictions === 'yes'`, but seemed cleaner to also declare it as false?